Answers.md

```diff
@@ -1,9 +1,18 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Describe some differences between ```display: block;``` and ```display: inline;```.
+Semantic HTML conveys _information_ to browsers/search engines. A Google robot will see a <title> or <h1> tag as containing information about the page contents, whereas <div> tags are only for structure within the page.
+
+2. Describe some differences between `display: block;` and `display: inline;`.
+
+a **block** element covers the entire line and will not share the line with other html elements (unless forced to), and an **inline** element can nestle next to other html elements on the same line.
 
 3. What are the 4 areas of the box model?
 
-4. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
+Content:Padding:Border:Margin ("Clowns Popping Balloon Monkeys")
+
+4. While using flexbox, what axis are you using when you use the property: `align-items: center`?
+   The Cross Axis
+
+5. What is the git command to commit staged changes as well as write a message?
 
-5. What is the git command to commit staged changes as well as write a message? 
\ No newline at end of file
+git commit -a -m "message"
```

Comment:
_**Comprehension Answers**_

Great and straightforward answers! Very clear and concise